packages/common-all/src/dnode.ts

```diff
@@ -1945,6 +1945,16 @@ export class SchemaUtils {
     return uri.fsPath.endsWith(".schema.yml");
   }
 
+  /**
+   * Adds leading zero if number is a single digit.
+   *
+   * Example: addPadding(3) -> "03"
+   *          addPadding(23) -> "23"
+   */
+  static addPadding(number: number) {
+    return number >= 0 && number < 10 ? "0" + number : "" + number;
```

Comment:
generally not cosure to add a "string" to a number without explicit casting since it can lead to unexpected bugs

also, since you are using luxon, there is native format strings which you can (and should) use

https://github.com/moment/luxon/blob/master/docs/formatting.md

Reply:
Gotcha, makes sense